File: RequestDTOs/BookRideRequest.cs
﻿namespace CarPooling.RequestDTOs
{
    public class BookRideRequest
    {
        public int UserId { get; set; }
        public string UserName { get; set; } = null!;
        public string OfferRideId { get; set; } = null!;
        public string StartPoint { get; set; } = null!;
        public string EndPoint { get; set; } = null!;
        public string Date { get; set; } = null!;
        public string TimeSlot { get; set; } = null!;
        public int Seats { get; set; } = 0;
        public string Price { get; set; } = null!;
    }
}

File: src/app/shared/models/book-ride-request.ts
export class BookRideRequest{
    startpoint: string;
    endPoint: string;
    date:string;
    timeSlot:string;
    
    constructor(startpoint: string,endPoint: string, date:string ,timeSlot:string
        ){
        this.startpoint=startpoint;
        this.endPoint = endPoint;
        this.date= date;
        this.timeSlot = timeSlot;
    }
}